src/date-parser/date-parser.mocha.ts

```diff
@@ -15,25 +15,19 @@
  * limitations under the License.
  */
 
-/// <reference path="../typings/mocha/mocha.d.ts" />
-/// <reference path="../typings/chai/chai.d.ts" />
-/// <reference path="../build/chronoshift.d.ts" />
-
 import { expect, assert } from "chai";
 var { deepEqual } = assert;
 
-declare function require(file: string): any;
-
-var chronoshift = require("../build/chronoshift");
-var Timezone = chronoshift.Timezone;
+import { Timezone } from '../timezone/timezone';
+import { parseISODate, parseInterval } from './date-parser';
 
-if (!chronoshift.WallTime.rules) {
-  var tzData:any = require("../lib/walltime/walltime-data.js");
-  chronoshift.WallTime.init(tzData.rules, tzData.zones);
+declare function require(file: string): any;
```

Comment:
it is necessary. I removed the node.js typings dependency (this is not a node.js bound project) so we need to keep the require type... it might be a good idea to move it into typings though